sa/_db-next/migrations/20171107160000_AddCertificateStatusIndexes.sql

```diff
@@ -0,0 +1,13 @@
+-- +goose Up
+ALTER TABLE certificateStatus
+       ADD INDEX `isExpired_ocspLastUpdated_Idx` (`isExpired`, `ocspLastUpdated`),
```

Comment:
Minor nit: The new indexes are cased `Idx`. The old indexes are cased `idx`. Looking at the existing migrations it seems like only `_db-next/migrations/20171015225800_AddRenewalBit.sql` and `_db/migrations/20150925184726_AddNamesTable.sql` use `Idx` and all of the other prev migrations use `idx`. Should we favour the latter and try to be consistent going forward? (I think at least one of the `Idx` instances was from me :blush: )